streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -456,6 +457,22 @@ public String consumers() {
         return consumerToPreviousStatefulTaskIds.keySet().toString();
     }
 
+    public Map<TaskId, Long> taskLagTotals() {
+        return taskLagTotals;
+    }
+
+    public SortedSet<TaskId> previousActiveTasks() {
+        return new TreeSet<>(previousActiveTasks.taskIds());
+    }
+
+    public SortedSet<TaskId> previousStandbyTasks() {
+        return new TreeSet<>(previousStandbyTasks.taskIds());
+    }
+
+    public SortedMap<String, Set<TaskId>> taskIdsByConsumer() {
```

Comment:
nit: call this `taskIdsByPreviousConsumer` or something to that effect (ie include the word "previous")

It's so hard to keep track of what pertains to the previous assignment vs the new assignment in this mess of a class lol. That's one of the biggest improvements in KIP-924